### PR TITLE
FE-924: properly handle empty auth token for auth requests

### PR DESCRIPTION
**Pull request by adubovsky**
@Glympse/web, PTAL.

### CODE REVIEW

File: app/src/lib/ajax.js [hunk 1]
```diff
@@ -185,12 +185,35 @@ define(function(require, exports, module)
 				attempts: ((retryOnError === false) ? 1 : MAX_ATTEMPTS),
 				retry: function()
 				{
-					// console.debug('[ajax] retry', options);
+					if (account && account.getToken())
+					{
+						addAuthData(authHeaderPrefix, account.getToken(), options);
```
Comment (layton-glympse) [inline app/src/lib/ajax.js]: Coule reduce this to one call to `account.getToken()`
```
var token = account.getToken();
if(account && token) {
    addAuthData(authHeaderPrefix, token, options);
}
```

**Comment by layton-glympse** [comment]
One comment, otherwise LGTM

File: app/src/lib/ajax.js [hunk 2]
```diff
@@ -154,19 +154,17 @@ define(function(require, exports, module)
...
-				var authHeaderPrefix = null;
 				if (auth.account)
 				{
 					account = auth.account;
-					// TRUE if not exact FALSE is passed
-					useHeader = (auth.useHeader !== false);
 
-					if (useHeader)
+					// TRUE if not exact FALSE is passed
+					if (auth.useHeader !== false)
```
Comment (tcolar) [inline app/src/lib/ajax.js]: I thought python uses `False` with a big 'F' ?

**Comment by j5kay** [comment]
LGTM

**Pull request merged**